### DISCUSSION TITLE
windows multi-session install

### DISCUSSION THREAD

**Discussion by grue-bleen** [Lumieducation/Lumi#2067] [Q&A]
we are trying to deploy lumi on our azure virtual desktop infrastructure.  for session host optimization reasons, we do not run microsoft store or updates on our virtual desktop session hosts.

the lumi windows installer exe seems to insist on installing itself into into %USERPROFILE%\AppData\Local\, which would cause a bit of an issue in our deployment with fslogix managed user profiles.

would there be any simple solution to locate common install files in ProgramData somehow?

**Comment by sr258** [comment]
We've got an enterprise installer on our roadmap, and I hope we'll come to implementing it in January. This will allows custom installation locations. How about temporary files? Where do you normally store them?

**Reply by grue-bleen** [reply]
%USERPROFILE%\AppData\Local\Temp would be the usual resting place for temp files, I believe.

**Reply by grue-bleen** [reply]
On the enterprise installer, is there an issue or roadmap item that we could track and voluntarily guinea pig for?
